infra/src/webapp.tf

```diff
@@ -225,7 +236,12 @@ module "webapp_functions_app_staging_slot" {
 
   always_on = "true"
 
-  app_settings = local.webapp_functions_app_settings
+  app_settings = merge(
+    local.webapp_functions_app_settings,
+    {
+      "AzureWebJobs.OnLegacyServiceChange.Disabled" = "1"
+    }
+  )
```

Comment:
Please note that this configuration is explicitly written both in the production and staging slots. This is because **listeners must be kept disabled in the staging slot, otherwise they will _steal_ items from queues/feeds**, being them listening to the same resources of production.

Alternatively, we can use dedicated queues/feeds for staging slots (that could be interesting).

Reply:
Ok clear, at the moment I would keep this double disabling